tools/create_operators_pull_request.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import os.path
+import sys
+import tempfile
+
+from utils import (
+    run,
+    clone_repository,
+    random_short_string,
+    configure_git_user,
+    create_pull_request,
+)
+
+PROGRAM_NAME = os.path.basename(__file__)
+
+log = logging.getLogger(__name__)
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%SZ",
+)
+
+
+KUDOBUILDER_OPERATORS_REPOSITORY = "kudobuilder/operators"
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(
+        description="Open a PR bringing a KUDO Operator's files to the "
+        + "kudobuilder/operators repository"
```

Comment:
Use the constant for the repo name? Here and elsewhere.

Reply:
- 8783916d19ba397e090daefdd8d734a72ecac2df
- 4480d59c43d844721d7aa2ad4c76707294566406